server/Gemfile

```diff
@@ -2,6 +2,8 @@ source 'https://rubygems.org'
 
 gemspec
 
+gem 'txgh', path: '../txgh'
```

Comment:
Yeah, this branch contains changes to both txgh and txgh-server. The tests won't pass without this line. I'll remove it once the new version of txgh is published.